clang/include/clang/Basic/Attr.td

```diff
@@ -3315,7 +3315,7 @@ def RequiresCapability : InheritableAttr {
   let TemplateDependent = 1;
   let ParseArgumentsAsUnevaluated = 1;
   let InheritEvenIfAlreadyPresent = 1;
-  let Subjects = SubjectList<[Function]>;
+  /*let Subjects = SubjectList<[Function]>;*/
```

Comment:
Instead of removing it, can this be updated to be correct (we have `FunctionPointer` as a subject)?